main/githooks.py

```diff
@@ -766,7 +766,7 @@ def check_commit_msg(message, files):
     does not contain required marker.
 
     '''
-    if re.match(r'^Merge branch \'.+?\' into .+', message):
+    if re.match(r'^Merge branch \'.+?\'( of github.com:[^\s]+)? into .+', message):
```

Comment:
I wonder about making this even more general, so the part in between the pair of ' and the "into" can be literally anything - so I could say "Merge branch 'x' blah blah blah into blah blah"

On an unrelated note, an you please see what form of message is produced for a cherry pick or backout and check these are also allowed.

Reply:
The commit message of a cherry pick is the same message the cherry-picked commit.